app/src/pages/[resourceType]/[id].tsx

```diff
@@ -73,47 +110,72 @@ export default function ResourceIDPage({ jsonData }: InferGetServerSidePropsType
             <Text size="xl" color="gray">
               {jsonData.resourceType}/{jsonData.id}
             </Text>
-            <Button w={240} loading={draftMutation.isLoading} onClick={createDraftOfArtifact}>
-              Create Draft of {jsonData.resourceType}
-            </Button>
+            <Group>
+              <Button
+                w={240}
+                id="btn"
```

Comment:
```suggestion
```
I think we can safely get rid of this since we aren't doing any sort of unit testing for this app where an id would need to be specified